core/arch/arch.c

```diff
@@ -85,6 +85,8 @@ byte *app_sysenter_instr_addr = NULL;
 static bool sysenter_hook_failed = false;
 #endif
 
+bool d_r_avx512_code_in_use = false;
```

Comment:
Should be inside `ifdef X86` I would think (or moved to ... I guess arch/x86/emit_utils.c)

Reply:
Done.